_posts/release/2018-02-27-fluo-yarn-1.0.0.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Apache Fluo YARN 1.0.0 released
+date: 2018-02-27 15:00:00 +0000
+version: fluo-yarn-1.0.0
+---
+
+This is the first release of the [Apache Fluo YARN launcher][fluo-yarn] which runs Fluo applications in
+[Hadooop YARN][YARN]. The code for launching in YARN was originally distributed with [Fluo][fluo]. It has been
+moved out to its [own project][fluo-yarn] as Fluo now support multiple ways for launching applications. Moving
```

Comment:
could drop `out`
could change `now support multiple ways for launching applications` to `has multiple application launchers`